src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,21 @@
+html,
+body,
+ul,
+li,
+button,
+input,
+textarea,
+a,
+p,
+h1,
+h2,
+h3 {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+}
```

Comment:
You dont need apply this styles to all tags, for example font you can specify just for body